README.md

```diff
@@ -16,4 +16,4 @@ Testing Branch for all commits
     ├── routes   
     ├── services  
     ├── tests  
-    └── views                     
\ No newline at end of file
+    └── views                     
```

Comment:
Nice adding this

Reply:
I didnt include package.json file because the file was deleted from my previous pr. I have now included it and reflected the changes you pointed out @shonubijerry

Reply:
Everything is fine now